projects/storefrontstyles/scss/theme/sparta/_variables.scss

```diff
@@ -54,7 +56,7 @@ $theme-colors: (
 );
 
 //type — https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L225
-$font-size-base: 1rem; //16px
+$font-size-base: 1rem !default; //16px
 // $h1-font-size: $font-size-base * 2.375; //38px
 $h1-font-size: $font-size-base * 1.85; //36px
 $h2-font-size: $font-size-base * 1.75; //28px
```

Comment:
@morganm58 

all that being said, please create a new branch with the proper conventions!